nengo_ocl/simulator.py

```diff
@@ -549,7 +553,10 @@ def run_steps(self, N, progress_bar=True):
                 self._plans.call_n_times(B)
                 self._probe()
                 N -= B
-                progress.step(n=B)
+                try:
+                    progress.total_progress.step(n=B)
+                except AttributeError:
+                    progress.step(n=B)
```

Comment:
The `except` path is usually expensive to execute I believe. It might be better to use an if statement to ensure good performance across versions?

Reply:
Probably pretty trivial compared to the simulation step time, but it can't hurt :smile: